### PR TITLE
feat: initContainer support

### PR DESCRIPTION
**Pull request by PabloHiro**

### CODE REVIEW

**Review by eshepelyuk — CHANGES_REQUESTED** [review]
`initContainers` should be list of objects, i.e. default empty value will be `[]`

File: test/linter/test.sh [hunk 1]
```diff
@@ -80,6 +80,11 @@ if [ $? -ne 0 ]; then
   exit 1
 fi
 
+helm lint . --strict --set registry.initContainers.dummy.image=foo
```
Comment (eshepelyuk) [inline test/linter/test.sh:83]: We need test case both for registry and sync. And also positive and negative case for both: positive when passing list of object as value and negative when passing schema violating value.

File: test/unit/initcontainer_registry_test.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+---
+suite: test registry init container
+templates:
+  - deployment-registry.yaml
+tests:
+  - it: no selector
```
Comment (eshepelyuk) [inline test/unit/initcontainer_registry_test.yaml:6]: selector ??

File: test/unit/initcontainer_sync_test.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
+---
+suite: test sync init container
+templates:
+  - deployment-sync.yaml
+tests:
+  - it: no selector
```
Comment (eshepelyuk) [inline test/unit/initcontainer_sync_test.yaml:6]: same about selector

**Review by eshepelyuk — CHANGES_REQUESTED** [review]

**Comment by PabloHiro** [comment]
> `initContainers` should be list of objects, i.e. default empty value will be `[]`

I am copying implementation from: 
https://github.com/tchiotludo/akhq/blob/dev/helm/akhq/templates/deployment.yaml#L52
Look at how they use it:
https://github.com/tchiotludo/akhq/blob/dev/helm/akhq/values.yaml#L80

If you prefer that I implement it as an array instead of a dict, I can go for it. Don't know what is the best practice to be honest. They use the dictionary key as the `name` field of the initContainer.

**Comment by eshepelyuk** [comment]
> > `initContainers` should be list of objects, i.e. default empty value will be `[]`
> 
> I am copying implementation from: https://github.com/tchiotludo/akhq/blob/dev/helm/akhq/templates/deployment.yaml#L52 Look at how they use it: https://github.com/tchiotludo/akhq/blob/dev/helm/akhq/values.yaml#L80
> 
> If you prefer that I implement it as an array instead of a dict, I can go for it. Don't know what is the best practice to be honest. They use the dictionary key as the `name` field of the initContainer.

Pleaese declare `initContainers` as a list of objects.

**Comment by eshepelyuk** [comment]
Thanks

Available in 3.1.0
https://github.com/eshepelyuk/apicurio-registry-helm/releases/tag/3.1.0

**Pull request merged**